src/main/java/org/folio/service/orders/CompositePoLineValidationService.java

```diff
@@ -76,6 +76,66 @@ private List<Error> validatePoLineFormats(CompositePoLine compPOL) {
     return Collections.emptyList();
   }
 
+  public List<Error> validatePoLineMaterial(CompositePoLine compPOL) {
+
+    CompositePoLine.OrderFormat orderFormat = compPOL.getOrderFormat();
+
+    if (orderFormat == P_E_MIX) {
+      return P_E_MixCheck(compPOL);
+    } else if (orderFormat == ELECTRONIC_RESOURCE) {
+      return electronicResourceCheck(compPOL);
+    } else if (orderFormat == CompositePoLine.OrderFormat.PHYSICAL_RESOURCE) {
+      return physicalResourceCheck(compPOL);
+    } else if (orderFormat == CompositePoLine.OrderFormat.OTHER) {
+      return otherResourceCheck(compPOL);
+    }
```

Comment:
how about to use switch cases here (java 17 style)

Reply:
yep, good idea, will do